disk/datadog_checks/disk/disk.py

```diff
@@ -65,6 +68,39 @@ def __init__(self, name, init_config, instances):
         self._compile_tag_re()
         self._blkid_label_re = re.compile('LABEL=\"(.*?)\"', re.I)
 
+        self.log.info("entering DISK check")
+    
+        if platform.system() == 'Windows':
+            self._manual_mounts = instance.get('create_mounts', [])
+            if not self._manual_mounts:
+                self.log.info("No manual mounts")
+            else:
+                self.log.info("Manual mounts: {}".format(len(self._manual_mounts)))
+                for manual_mount in self._manual_mounts:
+                    self.log.info("mm: {}".format(manual_mount))
+                    nr = win32wnet.NETRESOURCE()
+                    remote_machine = manual_mount.get('host')
+                    
+                    share = manual_mount.get('share')
+                    uname = manual_mount.get('user')
+                    pword = manual_mount.get('password')
+                    mtype = manual_mount.get('type')
```

Comment:
We should offer some info if these are missing, something like:
```python
for option in ["share", "user", "password"]:
    if option not in manual_mount:
        raise ConfigurationError("When specifying a manual mount you must provide a '%s'", option)
```

Reply:
at least username & password you may not need.  At least on my testing with NFS, I didn't need a username/password.